mozilla_general/README.md

```diff
@@ -4,17 +4,17 @@ This style guide provides the basic guidelines for translating Mozilla products.
 
 ## Content
 
-* [Fluency](general_L10n_SG.md#fluency)
-* [Meaning-based Translation](general_L10n_SG.md#meaning-based-translation)
-* [Terminology](general_L10n_SG.md#terminology)
-* [Text that should not be translated](general_L10n_SG.md#text-that-should-not-be-translated)
-* [Emphasis guidelines](general_L10n_SG.md#emphasis-guidelines)
+* [Fluency](#fluency)
+* [Meaning-based Translation](#meaning-based-translation)
+* [Terminology](#terminology)
+* [Text that should not be translated](#text-that-should-not-be-translated)
+* [Emphasis guidelines](#emphasis-guidelines)
 
```

Comment:
Should a section be included to mention Photon briefly? This would help translators understand the expected user experience with Mozilla products. Then, in community style guides, the methods for creating that experience can be outlined on a case-to-case basis.